assignments/prototypes.js

```diff
@@ -1,13 +1,42 @@
 /*
   Object oriented design is commonly used in video games.  For this part of the assignment you will be implementing several constructor functions with their correct inheritance hierarchy.
 
-  In this file you will be creating three constructor functions: GameObject, CharacterStats, Humanoid.  
+  In this file you will be creating three constructor functions: GameObject, CharacterStats, Humanoid.
 
   At the bottom of this file are 3 objects that all end up inheriting from Humanoid.  Use the objects at the bottom of the page to test your constructor functions.
-  
+
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
-  
+  function GameObject(attributes) {
+    this.createdAt = attributes.createdAt;
+    this.dimensions = attributes.dimensions;
+    this.name = attributes.name;
+  }
+
+  GameObject.prototype.destroy = function (name){
+    return `${this.name} was removed from the game.`;
+  }
+
+
+    function CharacterStats(attributes) {
+      GameObject.call(this, attributes);
+      this.healthPoints = attributes.healthPoints;
+    }
+
```

Comment:
Here is where you would set up inheritance with GameObject. Consider:

CharacterStats.prototype = Object.create(GameObject.prototype);